app/views/generic_files/_show_actions.html.erb

```diff
@@ -0,0 +1,49 @@
+<div id="show_actions">
+<h2 class="non lower">Actions</h2>
+    <p>
+      <%= render_download_link %>
+      <% if Sufia.config.analytics %>
+        &nbsp;|&nbsp;
+        <%= link_to "Analytics", sufia.stats_generic_file_path(@generic_file), id: 'stats' %>
+      <% end %>
+      <% if can? :edit, @generic_file %>
+          &nbsp;|&nbsp;
+          <% if @generic_file.processing? %>
+            <%= t('sufia.upload.processing') %>
+          <% else %>
+            <%= link_to "Edit", sufia.edit_generic_file_path(@generic_file) %>
+          <% end %>
+      <% end %>
+      <% if can?(:create, FeaturedWork) && @generic_file.public? %>
+        <% if FeaturedWork.can_create_another? && !@generic_file.featured?%>
+          &nbsp;|&nbsp;
+          <%= link_to "Feature", sufia.featured_work_path(@generic_file, format: :json), data: {behavior: 'feature'} %>
+        <% elsif @generic_file.featured? %>
+          &nbsp;|&nbsp;
+          <%= link_to "Unfeature", sufia.featured_work_path(@generic_file, format: :json), data: {behavior: 'unfeature-page'} %>
+        <% end %>
+      <% end %>
+    </p>
+    <p>Export to:
+      <%= link_to 'EndNote', sufia.generic_file_path(@generic_file, format: 'endnote') %>
+      &nbsp;|&nbsp;
+      <%= link_to 'Zotero', sufia.static_path('zotero'), {id: 'zoteroLink', name: 'zotero', class: 'lightboxLink'} %>
+      &nbsp;|&nbsp;
+      <%= link_to 'Mendeley', sufia.static_path('mendeley'), {id: 'mendeleyLink', name: 'mendeley', class: 'lightboxLink'} %>
+    </p>
+    <div>Permanent url:</div> 
+    <div><a href=<%=request.base_url + '/id/' + @generic_file.ark_id %>><%= request.base_url + '/id/' %><br/><%= @generic_file.ark_id %></a></div>&nbsp 
```

Comment:
We need to be able to handle items that don't yet have ARKs (I get a "no implicit conversion of nil into String" error now). Since the rake task to add ARKs will take a long time to run, we'll have a gap between deploying the new release and having ARKs for everything, so we need to handle this case. We should omit the 'Permanent url:" and the ARK itself. Also: seems better to use paragraphs rather than divs, to match the other "action" blocks above.